test/common/alerts_exist.go

```diff
@@ -530,6 +530,24 @@ func commonExpectedRules() []alertsTestRule {
 				"MultiAZPodDistribution",
 			},
 		},
+		{
+			File: NamespacePrefix + "rhsso-operator-rhsso-slo-availability-alerts.yaml",
+			Rules: []string{
+				"RHOAMRhssoAvailability5mto1hErrorBudgetBurn",
+				"RHOAMRhssoAvailability30mto6hErrorBudgetBurn",
+				"RHOAMRhssoAvailability2hto1dErrorBudgetBurn",
+				"RHOAMRhssoAvailability6hto3dErrorBudgetBurn",
+			},
+		},
+		{
+			File: NamespacePrefix + "user-sso-operator-user-sso-slo-availability-alerts.yaml",
+			Rules: []string{
+				"RHOAMUserSsoAvailability5mto1hErrorBudgetBurn",
+				"RHOAMUserSsoAvailability30mto6hErrorBudgetBurn",
+				"RHOAMUserSsoAvailability2hto1dErrorBudgetBurn",
+				"RHOAMUserSsoAvailability6hto3dErrorBudgetBurn",
+			},
```

Comment:
I think this would fail for RHMI e2e but pass with RHOAM e2e test due to the name is different depending on install.
https://github.com/integr8ly/integreatly-operator/blob/36b9deaa1e97db98a8d5268d00c13a797e822d6b/pkg/products/rhssouser/prometheusRules.go#L86

We should pass the installation name into this function and format the rule names similar to this but using the upper case name :thinking: 
https://github.com/integr8ly/integreatly-operator/blob/36b9deaa1e97db98a8d5268d00c13a797e822d6b/test/common/alerts_exist.go#L555-L564

Reply:
fix done, Thank you Kevin. Hope it's better now